modules/developer_manual/pages/general/backporting.adoc

```diff
@@ -129,3 +132,21 @@ xdg-open "https://github.com/$repository/pull/new/$targetBranch...$targetCommit"
 NOTE: This command opens the Pull Request and sets the target branch
 (in our example `stable10`) for the backport automatically.
 
+=== Backporting Alias
+
+If you want to integrate the script into your git commands, you can use a git alias. +
+Open the `~/.gitconfig` file with the editor of your choice and add the following:
```

Comment:
You can simplify the start of the sentence to just: "Open the `~/.gitconfig` file".